web/src/components/pages/datasets/metadata-viewer/types.ts

```diff
@@ -35,7 +35,9 @@ export type Age = {
 
 export type Gender = {
   '': number
-  male: number
-  female: number
-  other: number
+  male_masculine: number
+  female_feminine: number
+  intersex: number
+  transgender: number
```

Comment:
One is missing I believe:
```suggestion
  transgender: number
  'non-binary': number
```